.github/workflows/continuous-integration.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
         uses: eskatos/gradle-command-action@v1
         with:
-          arguments: verGJF build
+          arguments: verGJF build -x :code-coverage-report:build
```

Comment:
Due to the `jdk17-unit-tests` module, we cannot run the code coverage report on JDK 8 anymore.  It seems to still work on JDK 11, fortunately.